README.md

```diff
@@ -64,9 +64,8 @@ Put them in the following locations:
 - `concent-secrets/$cluster/nginx-storage-ssl.crt`
 - `concent-secrets/$cluster/nginx-storage-ssl.key`
 
-NOTE: These files are needed even if you disable SSL for a given cluster (by setting `nginx_proxy_ssl = false` or `nginx_storage_ssl = false` in `vars.yml`).
-If SSL is disabled for a cluster, the corresponding files can contain anything.
-They can even be empty.
+NOTE: These files are needed even if you allow connection without ssl for a given cluster (by setting `nginx_proxy_allow_http = true` or `nginx_storage_allow_http = true` in `vars.yml`).
+They can not be empty.
```

Comment:
This paragraph is no longer necessary. Remove it.

It was there because it's counter-intuitive that the cert files need to be provided even when SSL is disabled. That's no longer true.

Reply:
Done.